app/dashboard/templates/process_bounty.html

```diff
@@ -36,7 +36,7 @@
             {% include 'shared/zero_balance_error.html' %}
             {% include 'shared/unlock_metamask.html' %}
             <div class="col offset-md-1 col-md-10 d-flex align-items-center justify-content-center" id="primary_form">
-              <div class="d-flex justify-content-center flex-column mt-5 w-50" id="primary_subform">
```

Comment:
there is similar problem in kill_bounty.html

Reply:
Done  :P